ngraph/frontend/paddlepaddle/src/op/elementwise_add.cpp

```diff
@@ -25,8 +25,27 @@ namespace op {
 OutputVector elementwise_add (const NodeContext& node) {
     auto x = node.get_ng_input("X");
     auto y = node.get_ng_input("Y");
-    // TODO : resolve broadcast
-    return {std::make_shared<ngraph::opset6::Add>(x, y)};
+
+    auto axis = node.get_attribute<int>("axis");
+
+    auto x_shape = x.get_shape();
+    auto y_shape = y.get_shape();
+
+    if ((axis == -1) || (axis == x_shape.size() - 1) || (x_shape.size() == y_shape.size())) {
+        return {std::make_shared<ngraph::opset6::Add>(x, y)};
+    }
+    else {
+        auto broadcast_shape = Shape(x_shape.size(), 1);
+        int32_t idx = 0;
+        for(auto it = y_shape.begin(); it != y_shape.end(); ++it, idx++)
+            broadcast_shape[axis+idx] = *it;
+
+        std::cout << broadcast_shape << std::endl;
+
+        auto broadcast_shape_node = ngraph::opset6::Constant::create(ngraph::element::i64, ngraph::Shape{broadcast_shape.size()}, broadcast_shape);    
+        auto y_node = std::make_shared<ngraph::opset6::Reshape>(y, broadcast_shape_node, false);    
+        return {std::make_shared<ngraph::opset6::Add>(x, y_node)};             
+    }   
```

Comment:
This code is common for other eltwise ops (add, div), could you implement a common templated/parameterized with a simple inlined op factory snippet and call it here and in elementwise_div?